src/o365/tenant/commands/report/tenant-report-activeuserdetail.spec.ts

```diff
@@ -0,0 +1,117 @@
+import commands from '../../commands';
```

Comment:
File renamed from `tenant-report-....ts` to `report-....ts` to match the naming conventions